<launch>

  <!-- args corresponding to environment variables -->
  <arg name="UAV_NAME" default="$(env UAV_NAME)" />



  <group ns="$(arg UAV_NAME)">
    <node name="map_global_generation" pkg="mrs_ingeniarius_utils" type="map_global_generation" output="screen">

      <param name="uav_name" value="$(arg UAV_NAME)" />
	    <param name="uav_names" type="yaml" value="[uav1, uav2, uav3]"/>
      <param name="frame_output" value="common_origin" />
      <param name="wait_time" value="15.0" />
      <param name="save_pcd_file" value="true" />
      <param name="use_icp" value="true" />

      <param name="save_map_topic" value="save_map" />

    </node>

  </group>

</launch>

